### ISSUE TITLE
Possible issue: the middle button of the mouse is not working in the menu.

### ISSUE THREAD

**Issue by PonderFox0643** [SkytAsul/JukeBox#51]
I found this problem in spigot1.17.1 and paper1.20.1. If the game mode is survival mode, the mouse middle button cannot be used to add music, but only creative mode can.
So what happened?

**Comment by SkytAsul** [comment]
Yeah it took me ages to realize middle click only works in creative mod. Unfortunately I stopped working on JukeBox ^^'
The plugin is open source so you can ask any developer to modify the middle-click button and change it to another click...
